chartfx-chart/src/main/java/de/gsi/chart/XYChart.java

```diff
@@ -60,31 +61,30 @@
     private boolean callCanvasUpdateLater;
     private final ChangeListener<Side> axisSideChangeListener = this::axisSideChanged;
 
-    public XYChart() {
-        this(null, null);
-    }
-
     /**
      * Construct a new XYChart with the given axes.
      *
-     * @param xAxis the axis to use as primary x-Axis
-     * @param yAxis the axis to use as primary y-Axis
+     * @param axes All axes to be added to the chart
      */
-    public XYChart(final Axis xAxis, final Axis yAxis) {
-        super();
+    public XYChart(final Axis... axes) {
+        super(axes);
 
-        if (xAxis != null) {
-            if (xAxis.getSide() == null || !xAxis.getSide().isHorizontal()) {
-                xAxis.setSide(Side.BOTTOM);
+        for (int dim = 0; dim < axes.length; dim++) {
+            final Axis axis = axes[dim];
+            if (axis == null) {
+                continue;
             }
-            getAxes().add(xAxis);
-        }
-
-        if (yAxis != null) {
-            if (yAxis.getSide() == null || !yAxis.getSide().isVertical()) {
-                yAxis.setSide(Side.LEFT);
+            switch (dim) {
```

Comment:
Codacy found an issue: [A switch statement does not contain a break](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)